myriad-scheduler/src/main/java/org/apache/myriad/scheduler/MyriadOperations.java

```diff
@@ -251,13 +256,20 @@ public void shutdownFramework() {
       // Stop the driver, tasks, and executor.
       driverStatus = driverManager.stopDriver(false);
       LOGGER.info("Myriad driver was shutdown with status " + driverStatus);
-      try {
-        myriadWebServer.stop();
-      } catch (Exception e) {
-        LOGGER.info("Failed to shutdown Myriad webserver: " + e.getMessage());
-        return;
-      }
+    }
+
```

Comment:
Please include a log statement that reads something like "Received request to shutdown Myriad Framework.."